codec-http/src/main/java/io/netty/handler/codec/http/ServerCookieDecoder.java

```diff
@@ -1,182 +0,0 @@
-/*
- * Copyright 2014 The Netty Project
- *
- * The Netty Project licenses this file to you under the Apache License,
- * version 2.0 (the "License"); you may not use this file except in compliance
- * with the License. You may obtain a copy of the License at:
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
- * License for the specific language governing permissions and limitations
- * under the License.
- */
-package io.netty.handler.codec.http;
-
-import java.util.Collections;
-import java.util.Set;
-import java.util.TreeSet;
-
-import static io.netty.handler.codec.http.CookieEncoderUtil.*;
-
-/**
- * A <a href="http://tools.ietf.org/html/rfc6265">RFC6265</a> compliant cookie decoder to be used server side.
- *
- * Only name and value fields are expected, so old fields are not populated (path, domain, etc).
- *
- * Old <a href="http://tools.ietf.org/html/rfc2965">RFC2965</a> cookies are still supported,
- * old fields will simply be ignored.
- *
- * @see ServerCookieEncoder
- */
-public final class ServerCookieDecoder {
```

Comment:
This class was introduced in one of my previous PR. It has never been released since. It's moving to the cookie package.